README.md

```diff
@@ -39,13 +39,14 @@ For development, we suggest running the Docker conrtainer in interactive mode.
 To do so, you must mount the proper resources. 
 
 ```shell
-nvidia-docker run -it --rm -v $(pwd):/workspace/ gaussian_splatting 
+docker run -it --gpus all --rm -v $(pwd):/workspace/ gaussian_splatting 
 ```
 
 ### 4. Activate the Conda Environment.
 
 ```shell
-conda activate gaussian_activate
+conda activate gaussian_splatting
+pip install submodules/simple-knn
```

Comment:
Installing this package at build time fails, but it succeed inside the container. That's not the cleanest solution, but eh, it works!